src/mo_interpreter/interpret.ml

```
@@ -1000,7 +1000,10 @@ and interpret_block env decs ro (k : V.value V.cont) =
 and declare_dec dec : val_env =
   match dec.it with
   | ExpD _
-  | TypD _ -> V.Env.empty
+  | TypD _
+  | MixinD (_) -> V.Env.empty
+  | IncludeD (_, _, _) -> (* TODO *)
```

Might be better to just assert/fail rather than continue